### PR TITLE
Ru translite

### PR DESCRIPTION
**Pull request by webmastak**
Corrected.

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: xdg-desktop-portal-gtk master\n"
+"Report-Msgid-Bugs-To: https://github.com/flatpak/xdg-desktop-portal-gtk/"
+"issues\n"
+"POT-Creation-Date: 2018-11-16 15:18-0500\n"
+"PO-Revision-Date: 2019-05-30 10:06+0200\n"
+"Last-Translator: Andrew Sazonow <sazonow1980.@gmail.com>\n"
+"Language-Team: Russian <gnome-ru@gnome.org>\n"
```
Comment (AsciiWolf) [inline po/ru.po]: Sorry, the correct Language-team seems to be: `Language-Team: Русский <gnome-cyr@gnome.org>\n`.

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,242 @@
+# Russian translation for xdg-desktop-portal-gtk.
+# Copyright (C) 2019 xdg-desktop-portal-gtk's COPYRIGHT HOLDER
+# This file is distributed under the same license as the xdg-desktop-portal-gtk package.
+# Christian Kirbach <Christian.Kirbach@gmail.com>, 2016, 2018.
```
Comment (AsciiWolf) [inline po/ru.po]: Is Christian Kirbach really the Russian translation author?

File: po/ru.po (hunk 1 above)
Comment (webmastak) [reply po/ru.po]: Corrected

**Comment by webmastak** [comment]
Im corrtect file. Pull request.

**Comment by AsciiWolf** [comment]
Looks good, except the first two commits that are already merged and so are conflicting. If you can, try making a new branch from actual master, then cherry-pick the three new commits from the ru_translite branch into the new branch. Or let's wait for @matthiasclasen first, maybe it is possible to merge specific commits from GitHub PR, I am not sure.

**Pull request closed without merging**